generator/src/test/java/com/scottlogic/deg/generator/cucumber/AValid.feature

```diff
@@ -7,33 +7,33 @@ Background:
 
 Scenario: Running an 'aValid' request that includes a value of a string "ISIN" should be successful
     Given foo is a valid "ISIN"
-    And foo is in set:
-      | "GB0002634946"  |
-      | "US0378331005"  |
-      | "GB0002634947"  |
-      | "US0378331006"  |
-      | "US378331005"   |
-      | "US37833100598" |
-      | "G00002634946"  |
-      | "400002634946"  |
-      | "GBP002634946"  |
-      | "GB000263494Z"  |
-      | "0002634946GB"  |
+      And foo is in set:
+        | "GB0002634946"  |
+        | "US0378331005"  |
+        | "GB0002634947"  |
+        | "US0378331006"  |
+        | "US378331005"   |
+        | "US37833100598" |
+        | "G00002634946"  |
+        | "400002634946"  |
+        | "GBP002634946"  |
+        | "GB000263494Z"  |
+        | "0002634946GB"  |
     Then the following data should be generated:
       | foo            |
       | null           |
       | "GB0002634946" |
       | "US0378331005" |
-    And the following data should not be included in what is generated:
-      | foo             |
-      | "GB0002634947"  |
-      | "US0378331006"  |
-      | "US378331005"   |
-      | "US37833100598" |
-      | "G00002634946"  |
-      | "400002634946"  |
-      | "GBP002634946"  |
-      | "GB000263494z"  |
+      And the following data should not be included in what is generated:
+        | foo             |
+        | "GB0002634947"  |
+        | "US0378331006"  |
+        | "US378331005"   |
+        | "US37833100598" |
+        | "G00002634946"  |
+        | "400002634946"  |
+        | "GBP002634946"  |
+        | "GB000263494z"  |
 
 
   Scenario: Running an 'aValid' request that includes a value of a string "ISIN" should fail with an error message
```

Comment:
I thought this test should error because 'isin' not in caps is a completely invalid input for the aValid field. Much the same as `a valid 'nonsense'` would error as 'nonsense' is not accepted by the a Valid restriction.

Reply:
That makes sense, it's an invalid input.